app/charts/shared/chart-state.ts

```diff
@@ -177,14 +176,14 @@ export type TemporalXVariables = {
 
 export const useTemporalXVariables = (
   x: GenericField,
-  { dimensionsByIri }: { dimensionsByIri: ComponentsByIri }
+  { dimensionsByIri }: { dimensionsByIri: DimensionsByIri }
 ): TemporalXVariables => {
   const xDimension = dimensionsByIri[x.componentIri];
   if (!xDimension) {
     throw Error(`No dimension <${x.componentIri}> in cube!`);
   }
 
-  if (!isTemporalDimension(xDimension)) {
+  if (!isDataCubeTemporalDimension(xDimension)) {
```

Comment:
Why did we have to put "DataCube" everywhere in those methods ?

Reply:
That was a temporary solution to gradually replace "new" (DataCube) types with "old" types (without prefix). Probably wasn't super necessary, but I found it easier to navigate the changes then 👀